docs/manual/kinds/vr-cat9kv.md

```diff
@@ -20,12 +20,16 @@ The Q200 simulation has a limited featureset compared to the UADP simulation.
 
 ## Resource requirements
 
+The [[[ kind_display_name ]]] is a resource-hungry VM. When launched with the default settings, it requires the following resources:
```

Comment:
May be worth specifying by default is requires the UADPs minimum resources. The table just shows the recommended resources for each ASIC simulation. Q200 will use UADP's 4vCPU and 18432M RAM unless the user overwrites this.